tsl/src/bgw_policy/job.c

```diff
@@ -55,6 +55,29 @@
 
 #define REORDER_SKIP_RECENT_DIM_SLICES_N 3
 
+static void
+log_retention_boundary(int elevel, PolicyRetentionData *policy_data, const char *message)
+{
+	char *relname;
+	Datum boundary;
+	Oid outfuncid = InvalidOid;
+	bool isvarlena;
+
+	getTypeOutputInfo(policy_data->boundary_type, &outfuncid, &isvarlena);
+
+	relname = get_rel_name(policy_data->object_relid);
+	boundary = policy_data->boundary;
+
+	if (outfuncid != InvalidOid)
```

Comment:
```suggestion
	if (OidIsValid(outfuncid))
```

Just for better code readability